src/sql/workbench/contrib/notebook/browser/notebookThemingContribution.ts

```diff
@@ -20,19 +20,20 @@ export class NotebookThemingContribution extends Disposable implements IWorkbenc
 
 	private hookNotebookThemesAndConfigListener(): void {
 		if (this.configurationService) {
-			this.updateNotebookThemes();
+			this.updateNotebookThemes(false);
 			this._register(this.configurationService.onDidChangeConfiguration(e => {
+				let affectsResultsGrid = e.affectsConfiguration('resultsGrid');
 				if (e.affectsConfiguration(OVERRIDE_EDITOR_THEMING_SETTING)
-					|| e.affectsConfiguration('resultsGrid')) {
-					this.updateNotebookThemes();
+					|| affectsResultsGrid) {
+					this.updateNotebookThemes(affectsResultsGrid);
 				}
 			}));
 		}
 	}
 
-	private updateNotebookThemes() {
+	private updateNotebookThemes(affectsResultsGrid: boolean) {
```

Comment:
I'd suggest just naming this forceThemeUpdate or something more generic - it doesn't necessarily need to be tied to affectsResultsGrid